plugins/plogs.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: plogs
+spec:
+  version: v0.1.1
+  homepage: https://github.com/kha7iq/plogs
+  shortDescription: Retrieve and stream colorized logs from Kubernetes pods
+  description: |
+    The plogs plugin enables users to retrieve 
+    colorized logs from Kubernetes pods with 
+    various filtering and monitoring options.
```

Comment:
consider wrapping at 80 chars instead as we actually show this as-is to the users.

Reply:
This is also done